projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/secrets/service/vault/VaultJobSecretsService.java

```diff
@@ -23,6 +27,9 @@
 @ConditionalOnProperty(value = "featureflag.vault.integration.enabled")
 public class VaultJobSecretsService implements com.vmware.taurus.secrets.service.JobSecretsService {
 
+  @Value("${vdk.vault.size.limit}")
+  private int sizeLimit = 1048576;
```

Comment:
Please, always include the unit in a variable name - in this case I think it's `bytes` , right ? 
E.g "sizeLimitBytes"  and `vdk.vault.size.limit.bytes` or something along those lines. 

Future maintainer and administrators configuring it would be very grateful for this clarity.

Reply:
fixed.